docs: network pages update (#961)
oasisprotocol/docs: docs/dapp/cipher/network.mdx

---
id: network

> title: Network Information

Reviewer: (non-blocking) Our conventions appears to derive this header 1?

@matevz One of our [plugins](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs) auto generates this from the Markdown right?

Author: I removed all except description, as also discussed here https://github.com/oasisprotocol/sapphire-paratime/pull/415

> description: Network Information for Emerald

Reviewer: ```suggestion
description: Network Information for Cipher
```

[paratime-client-node]: ../../node/run-your-node/paratime-client-node.mdx

| Provider                    | Mainnet RPC URLs    | Testnet RPC URLs            |
|-----------------------------|---------------------|-----------------------------|
| [Oasis Protocol Foundation] | `grpc.oasis.io:443` | `testnet.grpc.oasis.io:443` |

[Oasis Protocol Foundation]: https://oasisprotocol.org

## Block Explorers

| Name/Provider               | Mainnet URL                               | Testnet URL                               | EIP-3091 compatible |
|-----------------------------|-------------------------------------------|-------------------------------------------|---------------------|
| Oasis Scan ([Bit Cat])                      | [https://www.oasisscan.com/paratimes/000…7cb](https://www.oasisscan.com/paratimes/000000000000000000000000000000000000000000000000e199119c992377cb) | [https://testnet.oasisscan.com/paratimes/000…000](https://testnet.oasisscan.com/paratimes/0000000000000000000000000000000000000000000000000000000000000000) | No |

[Bit Cat]: https://www.bitcat365.com/

:::tip

Only rudimentary block explorer features exist for Cipher. Consider debugging

> the Cipher transactions using the [`oasis paratime show`] command part of the

Reviewer: ```suggestion
Cipher transactions with the [`oasis paratime show`] command using the [Oasis CLI].
```
